examples/hybrid/package.json

```diff
@@ -0,0 +1,59 @@
+{
+    "private": true,
+    "name": "@theia/example-hybrid",
+    "version": "0.3.11",
+    "theia": {
+        "target": "hybrid"
```

Comment:
We should think about a better target name that explicitly points to a backend serving electron and browser clients.

Reply:
I liked `hybrid` because it was short and a mix of both `electron` and `browser` capable. If you have any idea I am all ears.